code/modules/research/stock_parts.dm

```diff
@@ -201,6 +201,18 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 		new /obj/item/stock_parts/micro_laser(src)
 		new /obj/item/stock_parts/matter_bin(src)
 
+/obj/item/storage/part_replacer/engi //used on echo
```

Comment:
A pre-loaded RPED? Exclusive on a map?

Reply:
Echo currently spawns with a few tier 3 parts on the thermo machine, this simply puts them inside an RPED and adds a few extra. Cargo can currently buy RPED on round start.